docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,417 @@
+key = "/home/sasha/scripts/event-handler/client.key"
+url = "https://localhost:8888/test.log"
+
+[teleport]
+addr = "mytenant.teleport.sh:443"
+identity = "identity"
+```
+
+To start the event handler, run the following command:
+
+```code
+$ ./teleport-event-handler start --config teleport-event-handler.toml
+```
+
+</TabItem>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+
+```toml
+storage = "./storage"
+timeout = "10s"
+batch = 20
+namespace = "default"
+
+[forward.fluentd]
+ca = "/home/sasha/scripts/event-handler/ca.crt"
+cert = "/home/sasha/scripts/event-handler/client.crt"
+key = "/home/sasha/scripts/event-handler/client.key"
+url = "https://localhost:8888/test.log"
+
+[teleport]
+addr = "teleport.example.com:443"
+identity = "identity"
+```
+
+To start the event handler, run the following command:
+
+```code
+$ ./teleport-event-handler start --config teleport-event-handler.toml
+```
+
+</TabItem>
+<TabItem scope={["oss", "enterprise"]} label="Helm Chart">
+
+Use the following template to create `teleport-plugin-event-handler-values.yaml`:
+
+```yaml
+eventHandler:
+  storagePath: "./storage"
+  timeout: "10s"
+  batch: 20
+  namespace: "default"
+
+teleport:
+  address: "example.teleport.com:443"
+  identitySecretName: teleport-event-handler-identity
+
+fluentd:
+  url: "https://fluentd.fluentd.svc.cluster.local/events.log"
+  sessionUrl: "https://fluentd.fluentd.svc.cluster.local/session.log"
+  certificate:
+    secretName: "teleport-event-handler-client-tls"
+    caPath: "ca.crt"
+    certPath: "client.crt"
+    keyPath: "client.key"
+
+persistentVolumeClaim:
+  enabled: true
+```
+
+To start the event handler in Kubernetes, run the following command:
+
+```code
+$ helm install teleport-plugin-event-handler teleport/teleport-plugin-event-handler \
+  --values teleport-plugin-event-handler-values.yaml \
+  --version (=teleport.plugin.version=)
+```
+
+</TabItem>
+</Tabs>
+
+<Admonition type="note">
+
+This example will start exporting from `May 5th 2021`:
+
+```code
+$ ./teleport-event-handler start --config teleport-event-handler.toml --start-time "2022-02-02T00:00:00Z"
+```
+
+The start time can be set only once, on the first run of the tool.
+
+If you want to change the time frame later, remove the plugin state directory
+that you specified in the `storage` field of the handler's configuration file.
+
+</Admonition>
+
+Once the handler starts, you will see notifications in Fluentd about scanned and forwarded events:
+
+```txt
+INFO[0046] Event sent id=0b5f2a3e-faa5-4d77-ab6e-362bca0994fc ts="2021-06-08 11:00:56.034 +0000 UTC" type=user.login
+...
+```
+
+The Logs view in Datadog should now report your Teleport cluster events:
+
+![Datadog Logs](../../../img/management/datadog-logs.png)
+
+Review the Fluentd output plugin for Datadog [README
+file](https://github.com/DataDog/fluent-plugin-datadog/blob/master/README.md)
+to learn how to customize the log format entering Datadog.
+
```

Comment:
It might be good to add a quick section about visualizing audit event counts by the "event" field, e.g., in the Splunk guide (https://goteleport.com/docs/management/export-audit-events/splunk/?scope=enterprise#step-44-visualize-your-audit-logs-in-splunk)

Reply:
It might be good, but it looks like that would require extra configuration to format the data so it could be separated by type, and I'd rather not go that deep into documenting a third party plugin written for another third party tool.

Reply:
As far as I know, Datadog should be automatically parsing the JSON documents the plugin is sending to it—is that not the case in this setup?

Reply:
Nope, doesn't seem to be.